firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -250,36 +210,81 @@ def import_job_dry_run(data, import_job_instance):
 
 
 @shared_task
-def batch_import(import_job_instance_id, dry_run=True):
-    # This method passes the provided CSV file through the clean method,
-    # does a dry-run or the import as per the dry_run flag provided,
-    # then parses through the result to return a validation report.
-
+def batch_import_dry_run(import_job_instance_id):
+    # Validates a provided CSV before importing provided entries
     logger = get_task_logger(__name__)
     task_id = current_task.request.id
     logger.info(
         ASYNC_TASK_START_TEMPLATE,
-        f"import_job_instance_id: {import_job_instance_id}, dry-run: {dry_run}",
+        f"import_job_instance_id: {import_job_instance_id}, dry-run: True",
     )
 
     import_job_instance = ImportJob.objects.get(id=import_job_instance_id)
 
-    if dry_run:
-        import_job_instance.validation_status = JobStatus.STARTED
-        import_job_instance.validation_task_id = task_id
-    else:
-        # we don't need to set the import_job_instance primary task status here
-        # as that is assigned in the @confirm view
-        import_job_instance.task_id = task_id
+    file = import_job_instance.data.content.open().read().decode("utf-8-sig")
+    data = tablib.Dataset().load(file, format="csv")
+
+    # Checks to ensure consistency
+    if import_job_instance.validation_status != JobStatus.ACCEPTED:
+        raise ValidationError(
+            "The specified job cannot be run due to consistency issues. "
+            "Please try using the validate endpoint to try again."
+        )
```

Comment:
In the async task, there will be nothing to catch this error. To handle this kind of failure condition in the task, I think we just need to set the validation_status to FAILED, log an info message, and exit.